src/eko/kernels/evolution_integrals.py

```diff
@@ -143,3 +146,192 @@ def j01_expanded(a1, a0, nf):
             integral
     """
     return j00(a1, a0, nf) - beta.b(1, nf) * j11_expanded(a1, a0, nf)
+
+
+@nb.njit("c16(f8,f8,u1)", cache=True)
+def j22_exact(a1, a0, nf):
+    r"""
+    NNLO-NNLO exact evolution integral.
+
+    .. math::
+        j^{(2,2)}(a_s,a_s^0) &=
+            \int\limits_{a_s^0}^{a_s}\!da_s'\,\frac{a_s'^3}
+                        {\beta_0 a_s'^2 + \beta_1 a_s'^3 + \beta_2 a_s'^4}
+            = \frac{1}{\beta_2}\ln\left(
+                    \frac{1 + a_s ( b_1 + b_2 a_s ) }{ 1 + a_s^0 ( b_1 + b_2 a_s^0 )}\right)
+                - \frac{b_1 \delta}{ \beta_2 \Delta} \\
+            \delta &= \atan \left( \frac{b_1 + 2 a_s b_2 }{ \Delta} \right)
+                    - \atan \left( \frac{b_1 + 2 a_s^0 b_2 }{ \Delta} \right) \\
+            \Delta &= \sqrt{4 b_2 - b_1^2}
```

Comment:
If I remember correctly you had a reference for this? Can you please add it (e.g. here and at the top of the module)

Reply:
The reference is cited in the docs, I haven't quoted them here because they don't report the result exactly this way, so I thought it was more confusing having it here.